.github/workflows/update-flake.yml

```diff
@@ -24,3 +24,4 @@ jobs:
         uses: DeterminateSystems/update-flake-lock@v8
         with:
           token: ${{ steps.generate-token.outputs.token }}
+          branch: update/flake-lock-${{ github.run_number }}
```

Comment:
Nest `github.run_number` will be 5, it's the monotonically increasing number from here:
https://github.com/http4s/http4s/actions/workflows/update-flake.yml